invenio/base/templates/_formhelpers_base.html

```diff
@@ -23,7 +23,7 @@
   {% set label_size = fieldkwargs.get('label_size') %}
   {% set field_size = fieldkwargs.get('field_size') %}
   {% set with_label = fieldkwargs.pop('with_label', True) %}
-  {%- if field.name == "csrf_token" or field.type == 'HiddenField' -%}
+  {%- if field.type == "CSRFTokenField" or field.type == 'HiddenField' -%}
```

Comment:
Maybe you can use https://flask-wtf.readthedocs.org/en/v0.8.3/#creating-forms

```python
<form method="POST" action=".">
    {{ form.hidden_tag() }}
```
to add all hidden fields (including CSRFTokenField)

Also to see if a field is hidden maybe better:

```python
field.widget.input_type == 'hidden'
```